round.test.js

```diff
@@ -0,0 +1,22 @@
+import test from 'ava';
+import round from './round';
+
+test('round to zero decimals', t => {
+    t.deepEqual(round(1.2345), 1);
+    t.deepEqual(round(1.2345, 0), 1);
```

Comment:
`t.is` may have better error messages. I am curious, why are you using `t.deepEqual`?

Reply:
copy paste! :)

Reply:
fixed